README.adoc

```diff
@@ -4,6 +4,20 @@
 :toc-title:
 :toclevels:
 
+[NOTE]
+====
+Each released `crc` binary expires 30 days after the release due to certificates embedded in the Openshift cluster.
+Using an older `crc` binary will result in an expired certificates error.
+
+To resolve this error:
+
+* link:https://github.com/code-ready/crc/releases[Download the latest CodeReady Containers release] and place the `crc` binary in your `$PATH`.
+* Remove the cluster with certificate errors using the `crc delete` command.
+* Set up the new release with the `crc setup` command.
+* Start the new cluster with the `crc start` command.
+
+====
```

Comment:
Something similar could be added to the documentation too (maybe in https://github.com/code-ready/crc/blob/master/docs/source/getting-started/content/proc_basic-troubleshooting.adoc ?)

Reply:
Absolutely, but we can do that in another PR. :)

Reply:
Yes I will put a PR for that doc update today.